src/components/Neighbors.cs

```diff
@@ -2,30 +2,25 @@
 
 public struct Neighbors : IEcsAutoReset<Neighbors>
 {
-    private EcsEntity[] _array;
+    public EcsEntity[] Array { get; set; }
```

Comment:
I think I'd like this to be a private set.
People should mutate this through the according functions.